test/unit/assignment_test.rb

```diff
@@ -186,11 +181,12 @@ class AssignmentTest < ActiveSupport::TestCase
           @assignment.allow_web_submits = false
           @assignment.save
           @target = Assignment.make({:allow_web_submits => false, :group_min => 1, :group_max => 1})
-          @target.create_groupings_when_students_work_alone
           # And for this test, let's make sure all groupings cloned have admin approval
-          @assignment.groupings.each do |grouping|
-            grouping.admin_approved = true
-            grouping.save
+          3.times do |i|
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.